patches/minecraft/net/minecraft/block/AbstractBlock.java.patch

```diff
@@ -144,12 +144,22 @@
        public AbstractBlock.Properties func_200941_a(float p_200941_1_) {
           this.field_200961_i = p_200941_1_;
           return this;
-@@ -933,7 +971,7 @@
+@@ -932,11 +970,17 @@
+          return this;
        }
  
++      @Deprecated // FORGE: Use the variant that takes a Supplier below
        public AbstractBlock.Properties func_222379_b(Block p_222379_1_) {
 -         this.field_222381_j = p_222379_1_.func_220068_i();
 +         this.lootTableSupplier = () -> p_222379_1_.delegate.get().func_220068_i();
```

Comment:
Should of been implemented as a `return lootFrom(p_222379_1_.delegate);` but whatever.